docusaurus.config.js

```diff
@@ -107,16 +107,16 @@ const config = {
         darkTheme: darkCodeTheme,
       },
       typesense: {
-        typesenseCollectionName: 'camino-docs',
+        typesenseCollectionName: 'camino-docs-default',
         typesenseServerConfig: {
           nodes: [
             {
-              host: 'docs.camino.foundation',
-              port: 8108,
+              host: 'typesense.camino.network',
+              port: 443,
               protocol: 'https',
             },
           ],
-          apiKey: 'K3s7dltmYhPw0oqOepeOx8liLZPRAGH2',
+          apiKey: 'TfCsn1wHrrpAzkHQP7747iPAS83Rsqn3',
```

Comment:
Isn't this something for secrets?

Reply:
Nope, this is only a read-only _search key_ and it goes into the client-side code. It is used to make search queries.

The one that goes into the secrets is another key that has write access to typesense collections. (`TYPESENSE_ADMIN_KEY` in repo's secrets)